api/envoy/config/route/v3/route_components.proto

```diff
@@ -542,6 +542,8 @@ message RouteMatch {
     //
     // Expect the value to not contain ``?`` or ``#`` and not to end in ``/``
     string path_separated_prefix = 14 [(validate.rules).string = {pattern: "^[^?#]+[^?#/]$"}];
+
+    string path_template_match = 15 [(validate.rules).string = {min_len: 1}];
```

Comment:
Please add docs and link to the issue in the PR.

Reply:
Added docstrings for both new proto fields